src/mlpack/tests/feedforward_network_test.cpp

```diff
@@ -160,13 +161,15 @@ TEST_CASE("FFVanillaNetworkTest", "[FeedForwardNetworkTest]")
 {
   // Load the dataset.
   arma::mat trainData;
-  data::Load("thyroid_train.csv", trainData, true);
+    if (!data::Load("thyroid_train.csv", trainData))
```

Comment:
```suggestion
  if (!data::Load("thyroid_train.csv", trainData))
```
Style fix. The indentation for the ```if``` statement is not required.